.github/workflows/windows-msvc-cuda.yml

```
@@ -15,7 +15,6 @@ jobs:
       fail-fast: false
       matrix:
         config:
-        - {version: "10.2.89.20191206", name: "cuda102/release/shared", "mixed": "ON"}
         - {version: "latest", name: "cuda-latest/release/shared", "mixed": "OFF"}
```

just to make sure we test mixed-precision kernel compilation
```suggestion
        - {version: "latest", name: "cuda-latest/release/shared", "mixed": "ON"}
```